source/whats-new.txt

```diff
@@ -22,6 +23,28 @@ Learn what's new in:
 * :ref:`Version 3.7 <version-3.7>`
 * :ref:`Version 3.6 <version-3.6>`
 
+.. _version-4.7:
+
+What's New in 4.7
+-----------------
+
+New features of the 4.7 {+driver-short+} release include:
+
+- Running ``MongoClient.connect()`` is now optional when connecting to your MongoDB instance
+- Use Zstandard compression when communicating with your MongoDB instance
+- Add support for the ``maxConnecting`` connection option
+- Change stream documents now show your updated documents before and after MongoDB applied changes to them
+- Added support for new change stream fields related to cluster to cluster replication
```

Comment:
```suggestion
- Added support for new change stream fields for cluster replication
```

Reply:
I think "Cluster to Cluster Replication" is the product name (at least from what I could find in [internal docs](https://jira.mongodb.org/browse/DRIVERS-1975)). Is the name of the product is now "cluster replication"? If so, could you point me to some resources that document this change/use this name, I was not able to find any.

Reply:
Ah I see, I didn't realize it was a product name since it was all lowercase before